py/desispec/io/meta.py

```diff
@@ -178,6 +190,18 @@ def findfile(filetype, night=None, expid=None, camera=None,
         exists; otherwise it returns the normal read/write path.
     """
     log = get_logger()
+
+    comptype=compression()
+
+    if comptype=="GZ" :
+        compsuffix=".gz"
+    elif comptype=="NONE"  :
+        compsuffix=""
+    else :
+        raise KeyError(f"unknown compression type '{comptype}'")
```

Comment:
For consideration: how about making `compression` a longer more meaningful name and having it directly return the suffix to use rather than post-facto parsing the `comptype`, e.g. `compsuffix = get_compression_suffix()`?

Reply:
suggestion applied in last commit